<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIngleChatWindow</class>
 <widget class="QMainWindow" name="SIngleChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>morseChat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>                                                                                                                                                  </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="messageEdit">
      <property name="maximumSize">
       <size>
        <width>782</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="sendButton">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
